app/javascript/components/profile/metrics.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="profile-metrics">
+    <h3>Métricas de tiempo: Tiempo entre creación de PR y asignación de revisor</h3>
+    <div
+      v-if="beingFetched"
+      class="loading-icon"
+    />
+    <div
+      v-else-if="Object.keys(pullRequestsInformation).length === 0"
+    >
+    <span>No hay información para mostrar</span>
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected indentation of 6 spaces but found 4 spaces  vue/html-indent